util/fetch/src/client.rs

```diff
@@ -297,7 +297,7 @@ impl Response {
 
 	/// Returns `true` if response status code is successful.
 	pub fn is_success(&self) -> bool {
-		self.status() == reqwest::StatusCode::Ok
+		self.status().is_success()
```

Comment:
Why was this changed? AFAIR `is_sucess()` returns true for any 2xx status code and I think we should be more strict in that case.